lxd/device/nic_bridged.go

```diff
@@ -631,10 +631,20 @@ func (d *nicBridged) Start() (*deviceConfig.RunConfig, error) {
 	}
 
 	if d.inst.Type() == instancetype.VM {
+		mtuStr, ok := d.config["mtu"]
```

Comment:
Just a nit, but the commit message of `lxd/device/nic/bridged` would have been clearer to see it only affects bridge nic. Or updating the wording to specify its for bridge NIC run configuration.